fluentui_persona/src/main/java/com/microsoft/fluentui/tokenized/persona/AvatarPie.kt

```diff
@@ -0,0 +1,163 @@
+package com.microsoft.fluentui.tokenized.persona
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.requiredSize
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.text.BasicText
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.token.ControlTokens
+import com.microsoft.fluentui.theme.token.controlTokens.AvatarInfo
+import com.microsoft.fluentui.theme.token.controlTokens.AvatarTokens
+import com.microsoft.fluentui.theme.token.controlTokens.AvatarSize
+import com.microsoft.fluentui.theme.token.controlTokens.AvatarType
+import com.microsoft.fluentui.theme.token.controlTokens.SlicedAvatarSize
+
+private val SPACER_SIZE = 2.dp
+
+@Composable
+fun AvatarPie(
+    group: Group, size: AvatarSize, noOfVisibleAvatars: Int = 2, avatarTokens: AvatarTokens? = null
+) {
+    val avatarInfo = AvatarInfo(
+        size
+    )
+    val token = avatarTokens
+        ?: FluentTheme.controlTokens.tokens[ControlTokens.ControlType.AvatarControlType] as AvatarTokens
+    val avatarSize = token.avatarSize(avatarInfo)
+
+    if (noOfVisibleAvatars == 1) {
+        BasicText(
+            "Pie should have minimum 2 members visible",
```

Comment:
BasicText should not be the fallback for an avatarpie. if avatar is only 1 then show only avatar. or if the group is emtpy then return empty composable but a basic text cannot be returned for avatar

Reply:
fixed